var/spack/repos/builtin/packages/qt/package.py

```diff
@@ -78,6 +79,11 @@ class Qt(Package):
     # https://github.com/spack/spack/issues/9209
     patch('qt4-gcc-and-webkit.patch', when='@4')
 
+    # Fix build failure with newer versions of GCC
+    # https://bugreports.qt.io/browse/QTBUG-74196
+    patch('https://github.com/qt/qtscript/commit/97ec1d1882a83c23c91f0f7daea48e05858d8c32.patch', sha256='ae88481a3ff63ab058cf9da6f5ae4397a983903109d907fb2ce4fcf91f9ca5e6', working_dir='qtscript', when='@5.0:5.12 %gcc@8.3:')
```

Comment:
Please break into multiple lines.
Should we be downloading this patch and including in the PR?  Or is it better to leave it as-is?

Reply:
Done. IMO, it is fine to download it as we do in other packages.